src/Life/Main/Update.hs

```diff
@@ -7,16 +7,32 @@ module Life.Main.Update
 import Path (Path, Rel, dirname, filename, toFilePath)
 import Path.IO (doesDirExist, doesFileExist, getHomeDir, makeRelative, resolveDir, resolveFile)
 
-import Life.Configuration (LifeConfiguration, parseGlobalLife, singleDirConfig, singleFileConfig,
-                           writeGlobalLife)
-import Life.Github (updateDotfilesRepo)
+import Life.Configuration (LifeConfiguration, lifePath, parseGlobalLife, singleDirConfig,
+                           singleFileConfig, writeGlobalLife)
+import Life.Github (Owner (..), updateDotfilesRepo)
+import Life.Main.Init (lifeInit)
+import Life.Message (chooseYesNo, errorMessage, infoMessage, promptNonEmpty, skipMessage,
+                     warningMessage)
+import Life.Shell (relativeToHome)
 
 -- | Add path to existing life-configuration file.
 lifeAdd :: FilePath -> IO ()
 lifeAdd path = do
-    -- TODO: check for .life existence
     -- TODO: check for dotfiles existence
```

Comment:
Can you please guide me through this one?

Reply:
Need to think... Not an easy problem.